src/player.ts

```diff
@@ -1,5 +1,6 @@
 // Instantiate the PLAYER variable in the Webflow page's head since it is needed across multiple scripts.
 PLAYER = document.querySelector('[data-element=audio-player]')
+PLAYER.addEventListener('pause', resetControllers)
```

Comment:
I should have noticed that before, but I really thought this was a constant (therefore a fairly simple object type), with the all caps, till I realized it was actually the player instance. Can be confusing

Reply:
Good flag, I think I must have let this carryover from the original script in Webflow. Will update to make it clearer!